modules/gfaffix/main.nf

```diff
@@ -0,0 +1,33 @@
+process GFAFFIX {
+    tag "$meta.id"
+    label 'process_medium'
```

Comment:
Does this need `process_medium`? It is is only currently using one core, does it need more than 6Gb RAM?

Reply:
Dependent on the input graph it might need over 100Gb RAM. But it only uses 1 thread.

Reply:
But for small graphs 1Gb suffices. It highly depends on the pangenome graph.

Reply:
process_medium is allocated 36Gb by default on the first attempt. Is that likely to work on "typical" input, or would it need more?

Reply:
I am sorry to be so vague, but it really depends on the input. We can build pangenome graphs from HLA genes https://github.com/pangenome/pggb/tree/master/data/HLA [1Gb], from 7 yeast assemblies https://pggb.readthedocs.io/en/latest/rst/tutorials/sequence_partitioning.html#steps [10-20Gb], or from 94 human haplotypes https://github.com/human-pangenomics/hpp_pangenome_resources [~150-200Gb]. More complex plant genomes or more number of haplotypes would require more RAM. 

There is no typical input here. Maybe later there is a way to estimate the RAM requirements before running the process, but that's not possible at the moment.

Reply:
It would be good to be fairly conservative, the resource requests can always be increased in the pipelines. if it only uses one thread, I would suggest process_single

Reply:
I'm ok defaulting to `process_single`

Reply:
Cool! Once this is in, we can merge :)